scenarios/kubernetes_janitor.py

```diff
@@ -72,22 +72,22 @@ def clean_project(project, hours=24, dryrun=False):
     'kubernetes-scale', # Let it's up/down job handle the resources
 ]
 
-PR_PROJECTS = [
-    'k8s-jkns-pr-kubemark',
-    'k8s-jkns-pr-gce',
-    'k8s-jkns-pr-gci-gce',
-    'k8s-jkns-pr-gke',
-    'k8s-jkns-pr-gci-gke',
-    'k8s-jkns-pr-gci-kubemark',
-    'k8s-jkns-pr-bldr-e2e-gce-fdrtn',
-    'k8s-jkns-pr-gce-etcd3',
-    'k8s-jkns-pr-gci-bld-e2e-gce-fd',
-]
+PR_PROJECTS = {
+    'k8s-jkns-pr-kubemark':3,
+    'k8s-jkns-pr-gce':3,
+    'k8s-jkns-pr-gci-gce':3,
+    'k8s-jkns-pr-gke':3,
+    'k8s-jkns-pr-gci-gke':3,
+    'k8s-jkns-pr-gci-kubemark':3,
+    'k8s-jkns-pr-bldr-e2e-gce-fdrtn':24,
+    'k8s-jkns-pr-gce-etcd3':3,
+    'k8s-jkns-pr-gci-bld-e2e-gce-fd':24,
```

Comment:
Is this still a thing? :) We should get rid of it, nobody is paying attention to it.